src/components/chats/index.js

```diff
@@ -8,10 +8,11 @@ import { useGetConversationQuery } from 'services/conversation/conversation';
 import profile from 'assets/profile.svg';
 
 import './styles.scss';
+import { CHAT, HOME } from 'constants/constants';
```

Comment:
What do you think if we define all style imports in the last lines?

Reply:
You are right, I ordered the imports in all the files but I missed that one